.github/workflows/go-lint.yml

```diff
@@ -61,56 +61,15 @@ jobs:
           go-version: '${{ inputs.go_version }}'
 
       - name: 'Lint (default configuration)'
-        if: ${{ hashFiles('.golangci.yml') == '' && hashFiles('.golangci.yaml') == '' }}
+        if: ${{ hashFiles('.golangci.yml', '.golangci.yaml') == '' }}
         uses: 'golangci/golangci-lint-action@07db5389c99593f11ad7b44463c2d4233066a9b1' # ratchet:golangci/golangci-lint-action@v3
         with:
           args: |-
-            --enable=${{ join(fromJson('[
-              "asciicheck",
-              "bidichk",
-              "bodyclose",
-              "containedctx",
-              "depguard",
-              "dogsled",
-              "errcheck",
-              "errchkjson",
-              "errname",
-              "errorlint",
-              "exhaustive",
-              "exportloopref",
-              "forcetypeassert",
-              "godot",
-              "gofumpt",
-              "goheader",
-              "goimports",
-              "gomodguard",
-              "goprintffuncname",
-              "gosec",
-              "gosimple",
-              "govet",
-              "ifshort",
-              "ineffassign",
-              "makezero",
-              "noctx",
-              "nolintlint",
-              "prealloc",
-              "predeclared",
-              "revive",
-              "sqlclosecheck",
-              "staticcheck",
-              "stylecheck",
-              "tenv",
-              "thelper",
-              "tparallel",
-              "typecheck",
-              "unconvert",
-              "unused",
-              "whitespace",
-            ]'), ',') }} --max-issues-per-linter=0 --max-same-issues=0 --timeout=5m
+            --config "${GITHUB_ACTION_PATH}/.golangci.yml"
```

Comment:
It doesn't look like `${{ github.acton_path }}` or `${GITHUB_ACTION_PATH}` is set...